hail/src/main/scala/is/hail/annotations/aggregators/RegionValueApproxCDFAggregator.scala

```diff
@@ -346,14 +349,26 @@ class ApproxCDFCombiner[@specialized(Int, Long, Float, Double) T: ClassTag : Ord
       lvl += 1
     }
 
+    lvl = 0
```

Comment:
it looks like you're adding both compaction counts to `mergedCompactionCounts(0)` twice here; is that intentional?

Reply:
Oops, no, good catch!